gradio/components.py

```diff
@@ -692,6 +694,7 @@ def __init__(
             interactive: if True, slider will be adjustable; if False, adjusting will be disabled. If not provided, this is inferred based on whether the component is used as an input or output.
             visible: If False, component will be hidden.
             elem_id: An optional string that is assigned as the id of this component in the HTML DOM. Can be used for targeting CSS styles.
+            randomize: If True, the value of the slider when the app loads is taken uniformly at random from the range given by the minimum and maximum.
```

Comment:
Can we update the docstring for `value` parameter to be:

"default value for slider. Ignored if `randomize=True`